language/golang/golang.go

```diff
@@ -59,6 +59,13 @@ func (l *Language) Files(log *log.Logger, repositoryPath string) (filePaths []st
 	return filePaths, nil
 }
 
+// TestFilePath returns the file path of a test file given the corresponding file path of the test's source file.
```

Comment:
Sounds complicated, why not?

-> `TestFilePath returns the test file path of a implementation file path.`